recipes/newrelic/infrastructure/alerts/golden.yml

```diff
@@ -0,0 +1,348 @@
+name: alerts-golden-signal
+displayName: Golden Signal Alerts
+description: New Relic install recipe for golden signal alerts
+repository: https://github.com/newrelic/newrelic-cli
+
+installTargets:
+  - type: host
+    os: linux
```

Comment:
Would we also want to offer this for Windows?

Reply:
Yes, once it's live an we get some feedback, I'll craft the windows version